exercises/node-javascript/encryption-context-start/store.js

```diff
@@ -0,0 +1,83 @@
+const {
```

Comment:
@mattsb42-aws @lizroth Here is the start store file.  The Encryption context needs to be bound to the message.